packages/block-editor/src/components/inserter/search-results.js

```diff
@@ -46,10 +66,22 @@ function InserterSearchResults( {
 	shouldFocusBlock = true,
 	prioritizePatterns,
 	selectBlockOnInsert,
-	orderInitialBlockItems,
 } ) {
 	const debouncedSpeak = useDebounce( speak, 500 );
 
+	const { prioritizedBlocks } = useSelect(
+		( select ) => {
+			const blockListSettings =
+				select( blockEditorStore ).getBlockListSettings( rootClientId );
+
+			return {
+				prioritizedBlocks:
+					blockListSettings?.prioritizedInserterBlocks || [],
```

Comment:
```suggestion
					blockListSettings?.prioritizedInserterBlocks || EMPTY_ARRAY,
```

It's preferred to return stable references.